towhee/utils/np_format.py

```diff
@@ -23,7 +23,7 @@ class NumpyArrayEncoder(json.JSONEncoder):
     """
     def default(self, o):
         if isinstance(o, np.ndarray):
-            return _NumpyFormat(o).to_dict()
+            return NumpyFormat(o).to_dict()
```

Comment:
NumpyArrayEncoder and NumpyArrayDecoder may be useless.

Reply:
I'll delete them